### PR TITLE
Datastore TTLS

### PR DESCRIPTION
**Pull request by LindsayHill**
Added section on using TTLs with Datastore items. Overhauled Datastore doc, tidied up some language and formatting.

### CODE REVIEW

**Comment by LindsayHill** [comment]
Updated docs to include notes on accessing encrypted values from python content. Addresses https://github.com/StackStorm/st2docs/issues/290

File: docs/source/datastore.rst [hunk 1]
```diff
@@ -74,30 +80,32 @@ Delete an existing key value pair:
...
-This basically means every user  has access to these variables. You'd typically use the
-jinja expression ``{{st2kv.system.key_name}}`` to refer to these variables in actions or
-workflows. Note that until v2.1, the scope used to be called ``system`` and therefore
-you'd have used jinja expression ``{{st2kv.system.key_name}}``.
-Starting version 1.5 of |st2|, you can now scope variables to a specific
+By default, all items stored in key-value store are stored in ``st2kv.system`` scope.
+This means every user has access to these variables. Use the Jinja expression
+``{{st2kv.system.key_name}}`` to refer to these variables in actions or workflows.
+Prior to v2.0.1, the scope was called ``system`` and therefore the Jinja expression
+was ``{{st2kv.system.key_name}}``. As of v2.2, this is no longer supported.
```
Comment (Kami) [inline docs/source/datastore.rst]: I assume this second should be just ``{{system.key_name}}``

**Comment by Kami** [comment]
One small typo, besides that, LGTM.

Nice work :+1:

File: docs/source/datastore.rst (hunk 1 above)
Comment (LindsayHill) [inline docs/source/datastore.rst]: Yep, thanks for spotting that

**Pull request merged**